packages/lambda-shared/src/api-path.ts

```diff
@@ -0,0 +1,40 @@
+export interface ActionData {
+    version: string;
+    action: string;
+    rest: string[];
+    urlPath: string;
+}
+
+export interface TileData {
+    x: number;
+    y: number;
+    z: number;
+    ext: string;
+}
+
+export const extractAction = (info: ActionData): void => {
```

Comment:
this function should not modify the input object, we should keep the input object clean and return the updated data.

being named `extract` implies that it would extract data not modify it.

Reply:
Are you say in general you rather copy instead of update or are you just pointing out the function name/action mismatch?